event-land.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>EventLand</title>
+
+    <meta
+      name="viewport"
+      content="initial-scale=1.0, user-scalable=no, width=device-width"
+    />
+    <link rel="stylesheet" href="css/style-eventland.css" />
+  </head>
+
+  <body>
+    <div class="container center-block">
+      <div class="top-bar">Top bar</div>
+
+      <nav class="main-navigation">Navigation</nav>
+      <header class="hero-header">Hero Header</header>
+      <main>
+        <section class="about-section">About</section>
+        <section class="features-section">Features</section>
+        <section class="advantage-section">Advantage</section>
+        <section class="testimonial-section">Testimonial</section>
+        <section class="cta-section">CTA</section>
+      </main>
```

Comment:
Based on out discussion, code convention, it's better to change it like this :) what do you think?!
```
<h1>dfsdfsdf</h1>
```